ios/RNGestureHandler.xcodeproj/project.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It was deleted in the `upgrade-helper`

Reply:
Removed in https://github.com/software-mansion/react-native-gesture-handler/pull/2209/commits/062af7ce45ef82ada9cd4c96275746d8cb2dff1f